robocop/checkers/lengths.py

```diff
@@ -323,17 +323,7 @@ def check_if_empty(self, node):
     def visit_SettingSection(self, node):  # noqa
         self.check_if_empty(node)
 
-    def visit_VariableSection(self, node):  # noqa
-        self.check_if_empty(node)
-
-    def visit_TestCaseSection(self, node):  # noqa
-        self.check_if_empty(node)
-
-    def visit_KeywordSection(self, node):  # noqa
-        self.check_if_empty(node)
-
-    def visit_CommentSection(self, node):  # noqa
-        self.check_if_empty(node)
+    visit_VariableSection = visit_TestCaseSection = visit_KeywordSection = visit_CommentSection = visit_SettingSection
```

Comment:
Is there maybe general visit_Section? VisitorChecker is ModelVisitor so it there is chance it matched groups too.

Reply:
I will look into it

Reply:
Updated with `visit_Section` (https://robot-framework.readthedocs.io/en/latest/autodoc/robot.parsing.model.html#robot.parsing.model.blocks.Section)